apps/api/src/auth/auth.service.ts

```diff
@@ -57,7 +64,12 @@ export class AuthService {
     }
   }
 
-  async login(email: string, password: string): Promise<UserEntity> {
+  async login(
+    email: string,
+    password: string
+  ): Promise<
+    Partial<UserEntity> & { refresh_token: string; access_token: string }
```

Comment:
@PhamAnhHoang maybe `{ refresh_token: string; access_token: string }` should not be standalone and should be implemented in `UserEntity` and `UserInterface`?